### DISCUSSION TITLE
logtopg metadata-generation-fails

### DISCUSSION THREAD

**Discussion by Faakhir30** [destrex271/pgweb-testing-harness#28] [Q&A] [answered]
I have tried installing using pip -r req.txt
I have tried updating pip, setuptools, wheel
 I've tried installing logtopg separately from the rest of the packages in Venv and it is failing with the following error in all cases.
 I've searched across the web but unfortunately couldn't find a solution.

 Any solution or recommendation? @destrex271 
 
 
![image](https://github.com/destrex271/pgweb-testing-harness/assets/110815427/2855c39e-c6f1-40d2-8de9-7f0eebe0bd0f)

**Comment by destrex271** [comment]
If you are trying to setup the project by installing the requirements it should work. Can you post the output of `pip install -r requirements.txt`?

**Reply by Faakhir30** [reply]
sure, ive recreated venv for this try, same response:

![image](https://github.com/destrex271/pgweb-testing-harness/assets/110815427/5e64eee1-fa76-4a49-98b9-e2d1ebce7d2b)

**Reply by destrex271** [reply]
You can remove logtopg from the requirements since it is not being used as such.
Also if you face problems even after that with regards to psycopg2, install the postgres development packages according to your distro/os.
Before removing logtopg also try to update pip and install wheel.

**Reply by Faakhir30** [reply, marked as answer]
Removing logtopg worked for me, thanks.

**Reply by Faakhir30** [reply]
I am currently looking forward to contributing to projects for my GSOC'24 application. 
Can you please confirm whether you are interested in enhancing this project in GSOC'24 as mentioned in the project ideas for this year?

![image](https://github.com/destrex271/pgweb-testing-harness/assets/110815427/830ab6a5-aedb-464c-a8a1-6c7aa0d54b42)

**Reply by destrex271** [reply]
Stay updated with the wiki page for now. I won't be mentoring the project but the mentor list will be updated soon. 
Also please keep these discussions on the concerned channels in slack. The repository discussions section is purely for technical discussions.
